resources/views/fields/upload.blade.php

```diff
@@ -18,14 +18,26 @@
         </div>
     @endif
 
-    {{-- Show the file picker on CREATE form. --}}
-    <input
-            type="file"
-            id="{{ $field['name'] }}_file_input"
-            name="{{ $field['name'] }}"
-            value="{{ old($field['name']) ? old($field['name']) : (isset($field['value']) ? $field['value'] : (isset($field['default']) ? $field['default'] : '' )) }}"
-            @include('crud::inc.field_attributes', ['default_class' =>  isset($field['value']) && $field['value']!=null?'form-control hidden':'form-control'])
-    >
+    <div class="js-parent-input">
+        @if (isset($field['value']) && $field['value'] !== null)
```

Comment:
No need of `$field['value'] !== null` :

http://php.net/manual/en/function.isset.php
> Determine if a variable is set and is not NULL.

And i would have used `!empty` instead of `isset`, what do you think ?

Reply:
Of course ! Isset was in the initial code and I've let it. I will replace it.

Reply:
:+1: